# HG changeset patch
# Parent 5d2033d8a6203c713b16fb2c3cd4e3d8b22ad3e3
Убрал проверку контрольной суммы и последовательности сообщений

diff -r 5d2033d8a620 -r 144a233ca358 block_class/udp_exchange.hpp
--- a/block_class/udp_exchange.hpp	Пт. апр. 17 09:32:36 2015 +0300
+++ b/block_class/udp_exchange.hpp	Вт. июля 21 12:49:56 2015 +0300
@@ -215,16 +215,16 @@
 				this->add_error_msg(
 					"Нарушена последовательность сообщений" );
 				_i_counter = m[1];
-				continue;
+				//continue;
 			}
 			//	Проверяем контрольную сумму
-			auto cs = * reinterpret_cast< const uint16_t * >( m.data() + 2 );
+/*			auto cs = * reinterpret_cast< const uint16_t * >( m.data() + 2 );
 			if( calc_control_sum< uint16_t >( m.data() + 4, m.size()-4 ) != cs )
 			{
 				this->add_error_msg(
 					"Принят буфер с неверной контрольной суммой" );
 				continue;
-			}
+			}*/
 			//	Считываем переменные
 			for( const auto & v: buf_desc->second.vars ) {
 				const auto v_begin = std::begin( m ) + v.offset();
